spec/models/spree/order_spec.rb

```diff
@@ -303,6 +303,53 @@
     end
   end
 
+  context "#cancel" do
```

Comment:
I think that `describe` is more apt for a method. `context` is better for the surroundings of the code under test. Your blocks below are a good examples for that.